db/schema.rb

```diff
@@ -28,6 +28,7 @@
     t.string "title", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.boolean "private", default: true
```

Comment:
Try to avoid creating booleans that allow nulls. You have what we call a three-state boolean. It can be true, false or null.

Reply:
That would just be refactored with a merge as `t.boolean "private", default: true, null: false`, correct? Default and Null don't need to be ordered in a particular way right?

Reply:
That's correct! :)